res/linux/mixxx-usb-uaccess.rules

```diff
@@ -12,6 +12,9 @@
 
 # Allen + Heath Ltd.
 SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="22f0", TAG+="uaccess"
+# Allen + Heath Xone 23C hardware mixer with USB audio interface
+# This is required so all 4 output channels of the audio interface are available.
```

Comment:
Please add the full information as a comment

> ...so this shows up as a 4 input/4 output soundcard - otherwise shows up as only 2 input/2 output.

Reply:
Ping

Reply:
I'm actually not sure if it shows up with 2 inputs or 4 inputs without this.

Reply:
Without this it only shows up as 2 inputs and 2 outputs (i.e you can connect only a single turntable)